### PR TITLE
Trello 041 outcome alert layout

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**
image

### CODE REVIEW

File: src/server/case/overview.njk [hunk 1]
```diff
@@ -2,17 +2,14 @@
 {% set pageName = 'Overview' %}
 {% from '_join.njk' import join %}
 {% from 'components/summary-card.njk' import appSummaryCard %}
-
 {% block page %}
-{% if appointmentsWithoutAnOutcome.length %}
+{% if appointmentsMissingOutcome.count %}
     {% set outcomeAlert %}
-        <h2 class="govuk-heading-m">You need to record an outcome for an appointment</h2>
+        <h2 class="govuk-heading-m">You have appintments that need attention</h2>
```
Comment (electrified) [inline src/server/case/overview.njk:8]: typo: appintments -> appointments

File: src/server/case/case.controller.ts [hunk 2]
```diff
@@ -50,10 +50,7 @@ export class CaseController {
         conviction,
         complianceFilter: ActivityComplianceFilter.WithoutOutcome,
       })
-      appointmentsWithoutAnOutcome = contacts.content
-        .flatMap(x => x.entries)
-        .filter(x => x.start < today)
-        .sort((a, b) => (a.start > b.start ? 1 : -1))
+      appointmentsWithoutAnOutcome = contacts.content.flatMap(x => x.entries).filter(x => x.start < today)
```
Comment (electrified) [inline src/server/case/case.controller.ts:53]: The count could be done here

**Review by electrified — CHANGES_REQUESTED** [review]

File: src/server/case/overview.njk (hunk 1 above)
Comment (dnyanesh-sonwane) [reply src/server/case/overview.njk]: that's silly typo :-( 
Fixed

File: src/server/case/case.controller.ts (hunk 2 above)
Comment (dnyanesh-sonwane) [reply src/server/case/case.controller.ts]: fixed

**Review by electrified — APPROVED** [review]

**Pull request merged**